ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -95,30 +95,35 @@ public void run() {
         long startedAt = -1;
         // Make sure nothing escapes this run method and kills the metastore at large,
```

Comment:
Nit: This comment applies to stuff after line 102, so I'd move it there.

Reply:
fixed